modin/logging/config.py

```diff
@@ -120,25 +120,58 @@ def configure_logging():
     pandas_version = pkg_resources.get_distribution("pandas").version
     num_physical_cores = str(psutil.cpu_count(logical=False))
     num_total_cores = str(psutil.cpu_count(logical=True))
-    svmem = psutil.virtual_memory()
     logger.info(f"Modin Version: {modin_version}")
     logger.info(f"Pandas Version: {pandas_version}")
     logger.info(f"Physical Cores: {num_physical_cores}")
     logger.info(f"Total Cores: {num_total_cores}")
-    logger.info(f"Memory Total: {bytes_int_to_str(svmem.total)}")
-    logger.info(f"Memory Available: {bytes_int_to_str(svmem.available)}")
-    logger.info(f"Memory Used: {bytes_int_to_str(svmem.used)}")
 
     if LogMode.get() != "enable_api_only":
         mem_sleep = LogMemoryInterval.get()
+        mem_logger = configure_memory_logging(job_id, formatter)
         mem = threading.Thread(
-            target=memory_thread, args=[logger, mem_sleep], daemon=True
+            target=memory_thread, args=[mem_logger, mem_sleep], daemon=True
         )
         mem.start()
 
     __LOGGER_CONFIGURED__ = True
 
 
+def configure_memory_logging(job_id, formatter):
```

Comment:
This function is instantiating and configuring the logger, but it's also logging the initial memory messages. I think it's doing too much at once. I think it's actually ok to get rid of this function and inline its body in `configure_logging`. If you don't want to do that, I'd prefer calling it `get_memory_logger` and moving the initial log messages into `configure_logging`.